package.json

```diff
@@ -54,14 +54,15 @@
   },
   "license": "Apache-2.0",
   "dependencies": {
+    "@11ty/eleventy": "^0.12.1",
```

Comment:
Some dependencies are re-arranged in here because `npm`'s idea of alphabetical order apparently differs from the sorting that I used when I modified `package.json` last.